src/components/Card/CardText.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component, PropTypes } from 'react'
+import { View, Text } from 'react-native'
+import Uranium from 'uranium'
+import { Breakpoints, connectTheme } from '../../index'
+
+class CardText extends Component {
+  
+  static propTypes = {
+    children: PropTypes.node,
+    /**
+     * Wrapper component style.
+     */
+    style: PropTypes.oneOfType([
+      PropTypes.object,
+      PropTypes.array,
+    ]),
+    /**
+     * Text style.
+     */
+    textStyle: PropTypes.oneOfType([
+      PropTypes.object,
+      PropTypes.array,
+    ]),
+  };
```

Comment:
Props at bottom of file, like `CardText.propTypes`--see other components for example

This is so, proptypes can be in the same part of the file for both functional and stateful components.